### PR TITLE
[FIX] Collectible favorites

### PR DESCRIPTION
**Pull request by gantunesr**

### CODE REVIEW

File: app/reducers/collectibles/index.js [hunk 1]
```diff
@@ -51,7 +54,11 @@ const fiatOrderReducer = (state = initialState, action) => {
 			const { selectedAddress, chainId, collectible } = action;
 			const collectibles = getFavoritesCollectibles(state.favorites, selectedAddress, chainId);
 			const indexToRemove = collectibles.findIndex(
-				({ tokenId, address }) => tokenId === collectible.tokenId && address === collectible.address
+				({ tokenId, address }) =>
+					// TO DO: Remove after moving favorites to controllers.
+					(typeof tokenId === 'number'
```
Comment (sethkfman) [inline app/reducers/collectibles/index.js:59]: I would consider making this a utils function that can be used in all 3 places.

**Review by sethkfman — APPROVED** [review]
Consider my comment. I know this is a temporary fix but if it lingers it may be a nice mid-term solution.

File: app/reducers/collectibles/index.js (hunk 1 above)
Comment (gantunesr) [reply app/reducers/collectibles/index.js]: Done

**Review by sethkfman — APPROVED** [review]
LGTM

**Pull request merged**